modules/carto/src/layers/h3-tileset-2d.ts

```diff
@@ -66,12 +66,20 @@ function tileToBoundingBox(index: string): GeoBoundingBox {
 // similar
 // Relative scale factor (0 = no biasing, 2 = a few hexagons cover view)
 const BIAS = 2;
-export function getHexagonResolution(viewport): number {
+export function getHexagonResolution(viewport: {
+  zoom: number;
+  latitude: number;
+  tileSize: number;
```

Comment:
Let's not include this in the `viewport` object as it will be confusing, use a separate parameter

Reply:
Done!